src/App.vue

```diff
@@ -44,15 +47,14 @@ export default {
       const user = confirmLogin(userData, this.allUsers)
       if (user) {
         this.updateLogin()
-        this.currentUser = user.id
+        setUserId(user.id)
```

Comment:
I am a fan of changing the userID to a function so that you can call it wherever you need in the code rather than passing a bunch of props around.